pkg/proxy/docker/linux.go

```diff
@@ -49,6 +52,8 @@ func locateTruststore(dfs *dockerfs.Filesystem) (*dockerfs.File, error) {
 		// Expected Cert Dir:  /var/lib/ca-certificates/{openssl,pem}/
 		// NOTE: JKS file also needs to be regenerated at /var/lib/ca-certificates/java-cacerts.
 		return dfs.OpenAndResolve("/var/lib/ca-certificates/ca-bundle.pem")
+	case "kaniko":
+		return dfs.OpenAndResolve("/kaniko/ssl/certs/ca-certificates.crt")
```

Comment:
might be helpful to link through to any kaniko docs/code related to this

Reply:
Couldn't find official documentation so I just linked to a line in the dockerfile that creates this directory.